main.cpp

```diff
@@ -79,43 +98,102 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+bool eat(int pizzaSlice, char pizzaName)
+
+{
+  ignoreUnused(pizzaSlice, pizzaName);
+
+  return {};  
+}
 /*
  2)
  */
+void looper(int barSize, double sampleRate = 44.1f)
+
```

Comment:
This function name does not contain a verb.
Name functions what they do.
- [ ] revise